sdk/nodejs/src/platforms.ts

```diff
@@ -66,85 +66,77 @@ SYSTEM_TO_LIBRARY_PATH.set(
   `${PLATFORM_RASPBERRY_PI}/${ARM_CPU_CORTEX_A53}/pv_rhino.node`
 );
 SYSTEM_TO_LIBRARY_PATH.set(
-  `${SYSTEM_LINUX}/${ARM_CPU_CORTEX_A53}${ARM_CPU_64}`,
-  `${PLATFORM_RASPBERRY_PI}/${ARM_CPU_CORTEX_A53}${ARM_CPU_64}/pv_rhino.node`
+    `${SYSTEM_LINUX}/${ARM_CPU_CORTEX_A53}${ARM_CPU_64}`,
```

Comment:
2 spaces, same for rest of file. I think your editor adds  spaces automatically.

Reply:
Ah, probably those were tab characters and my editor is set to convert tabs to 4 spaces